### ISSUE TITLE
Issue with local files

### ISSUE THREAD

**Issue by franc-ist** [GiovanniMCMXCIX/DiscoSpotipy#4]
When playing a local file, spotify's api throws error code 400 "bad id", which closes the script, instead of passing

**Comment by franc-ist** [comment]
note to self: 

```
Traceback (most recent call last):
  File "spotify-rpc-client.py", line 240, in <module>
    rpc.loop.run_until_complete(rpc.run())
  File "/usr/lib/python3.6/asyncio/base_events.py", line 467, in run_until_complete
    return future.result()
  File "spotify-rpc-client.py", line 227, in run
    await self.detect_now_playing()
  File "spotify-rpc-client.py", line 213, in detect_now_playing
    timestamp = await run(d, paused=True)
  File "spotify-rpc-client.py", line 132, in run
    async with self.session.request('GET', data['item']['album']['href'], headers={'Authorization': f'Bearer {self.token}'}) as resp:
TypeError: 'NoneType' object is not subscriptable
```

**Comment by GiovanniMCMXCIX** [comment]
Fixed by #5
